docs/sources/shared/datasources/tempo-search-traceql.md

```diff
@@ -0,0 +1,103 @@
+---
+headless: true
+---
+
+<!--
+This file documents the Search query type for the Tempo data source. It is available as a public preview.
+
+This shared file is included in these locations:
+/grafana/docs/sources/datasources/tempo/query-editor/index.md
+
+If you make changes to this file, verify that the meaning and content are not changed in any place where the file is included.
+
+Any links should be fully qualified and not relative: /docs/grafana/
+ -->
+
+# Create TraceQL queries using Search
+
+{{% admonition type="Note" %}}
```

Comment:
```suggestion
{{% admonition type="note" %}}
```

Admonition types should be specified in lower case.

Reply:
Thank you. I had some trouble with the admonition rendering and was playing with it to see if it made a difference. I built it today and things seem to be okay now.